requirements.txt

```diff
@@ -44,6 +44,7 @@ Pygments==2.12.0
 pyparsing==3.0.9
 python-dateutil==2.8.2
 python-dotenv==0.20.0
+pywin32==304
```

Comment:
Are we using Windows API extensions for anything?
It looks like it's mostly for mouse control and getting screen information. It can also make beeping sounds.

I remember having to remove this line on my machine since PyWin32 isn't available.

Reply:
I don't know what it is or what thing I installed that also installed it... but it's removed for now!